Warning in example2...

/home/ed/pio3/ParallelIO/examples/c/example2.c:555:60: warning: passing argument 4 of ‘PIOc_def_var_chunking’ from incompatible pointer type [-Wincompatible-pointer-types]
      if ((ret = PIOc_def_var_chunking(ncid, 0, NC_CHUNKED, chunksize)))
                                                            ^~~~~~~~~
In file included from /home/ed/pio3/ParallelIO/examples/c/example2.c:29:0:
/home/ed/pio3/ParallelIO/src/clib/pio.h:655:9: note: expected ‘const MPI_Offset * {aka const long long int *}’ but argument is of type ‘size_t * {aka long unsigned int *}’
     int PIOc_def_var_chunking(int ncid, int varid, int storage, const PIO_Offset *chunksizesp);
         ^~~~~~~~~~~~~~~~~~~~~
[100%] Linking C executable example2

--- a/examples/c/example2.c
+++ b/examples/c/example2.c
@@ -98,8 +98,7 @@ char dim_name[NDIM][NC_MAX_NAME + 1] = {"timestep", "x", "y"};
 int dim_len[NDIM] = {NC_UNLIMITED, X_DIM_LEN, Y_DIM_LEN};
 
 /** Length of chunksizes to use in netCDF-4 files. */
-size_t chunksize[NDIM] = {2, X_DIM_LEN/2, Y_DIM_LEN/2};
-
+PIO_Offset chunksize[NDIM] = {2, X_DIM_LEN/2, Y_DIM_LEN/2};
 
 /** Number of MPE events. The start and stop of each event will be
  * tracked, and graphed. This value is used outside of HAVE_MPE ifdefs.*/

